docs/containers/ssh.md

```diff
@@ -7,7 +7,9 @@ MetaDescription: Connect via SSH to Docker engine running on a remote machine an
 ---
 # Connect to remote Docker over SSH
 
-We recommend using the Visual Studio Code [Remote-SSH extension](/docs/containers/choosing-dev-environment.md#remote-machine) to connect to a remote machine running Docker engine, but it also possible to connect to the remote Docker engine directly, using SSH tunneling.
+We recommend using the Visual Studio Code [Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) extension to connect to a remote machine running Docker engine. You can use the Remote - SSH and [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extensions together. You may review the steps in the [remote development documentation](/docs/remote/containers#_open-a-folder-on-a-remote-ssh-host-in-a-container).
```

Comment:
```suggestion
We recommend using the Visual Studio Code [Remote - SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) to connect to a remote machine running Docker engine. You can use the Remote - SSH and [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extensions together. You may review the steps in the [remote development documentation](/docs/remote/containers#_open-a-folder-on-a-remote-ssh-host-in-a-container).
```

Reply:
@bamurtaugh and @bwateratmsft , does this seem like a fine change? (super small nit I know)

Reply:
I agree it'd be nice to be consistent. I think it would be fine to leave as-is, or we should update the below reference to be:

> You can use Remote - SSH and [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) together.

since it wouldn't make sense to make the word `extensions` in the current sentence link to just Remote - Containers. Preference on leaving as-is, or updating both sections?

Reply:
My preference would be to leave it as-is, but I still don't feel strongly. :smile:

Reply:
Leave as is! Ready for Greg!